src/components/FeatureFlagRenderer.js

```diff
@@ -86,6 +93,17 @@ export default class FeatureFlagRenderer extends Component<Props, State> {
     });
   }
 
+  identify (ldClient: LdClientWrapperType) {
```

Comment:
Probably should add a test for this

Reply:
Definitely, still trying to get these flow errors resolved.